src/util/classes.ts

```diff
@@ -23,26 +24,68 @@ export interface Category {
   children?: Category[];
 }
 
-const COLOR_UNCAT = '#CCC';
+// https://colorhunt.co/palette/4802
+const COLOR_GREEN = '#96cd39',
+  COLOR_SUPER_GREEN = '#54e346',
+  COLOR_BRIGHT_GREEN = '#A8FC00',
+  COLOR_UNCAT = '#CCC',
+  COLOR_SLIGHTLY_YELLOW = '#f5ff65',
+  COLOR_ORANGE = '#ffba47',
+  COLOR_RED = '#ff5b44';
 
 // The default categories
 // Should be run through createMissingParents before being used in most cases.
 export const defaultCategories: Category[] = [
   {
-    name: ['Work'],
-    rule: { type: 'regex', regex: 'Google Docs|libreoffice|ReText' },
-    data: { color: '#0F0' },
+    name: ['Work', 'Writing'],
+    rule: { type: 'regex', regex: 'Google Docs|Google Sheets|libreoffice|ReText' },
+    data: { color: COLOR_GREEN },
+  },
+  {
+    name: ['Work', 'Writing'],
+    rule: {
+      type: 'regex',
+      select_keys: ['app'],
```

Comment:
So this is set, but there is no way to change it in the UI? 

All `select_keys` should probably be removed from the default categories until the UI supports it.

Reply:
Yes, that's right.

You can't set them via the UI, but you can import a config file which contains them. I think we should add `select_keys` to the default config since you can 'export, modify, import' and to mutate these keys _and_ it makes it more explicit how we are deciding which app is in use.

Reply:
But that's very confusing for the user. They have no indication there is even a `select_keys` for those categories, right? (unless they export and inspect)

I really don't want to add this specifier to the default rules without:

 1. UI support for it
 2. More thorough testing of these new "multi-rule" categories in the UI.
     - What happens in the UI if a category with multiple rules has children?
     - What happens when you rename a category with multiple rules? Or assign it to a different parent?
     - What happens to `color` keys when you have multiple rules?
     - Ideally, a category with multiple rules should be listed only once in the category tree, with an indication that there are multiple rules for this one category (without listing it as two seperate categories, since that's likely to get confusing).